pkg/cidata/template.go

```diff
@@ -19,6 +19,15 @@ var templateFS embed.FS
 
 const templateFSRoot = "cidata.TEMPLATE.d"
 
+type CACerts struct {
+	RemoveDefaults bool
+	Trusted        []Cert
+}
+
+type Cert struct {
+	Lines []string
+}
```

Comment:
Having a slice of lines to represent a cert seems slightly weird to me; that is a representation detail. I do see that it is convenient for the template processing though, so maybe it is fine.